packages/edit-post/src/components/editor-initialization/test/listener-hooks.js

```diff
@@ -99,17 +81,29 @@ describe( 'listener hook tests', () => {
 		} );
 	} );
 	describe( 'useBlockSelectionListener', () => {
+		const registry = createRegistry( mockStores );
+		const TestComponent = ( { postId } ) => {
+			useBlockSelectionListener( postId );
```

Comment:
We could potentially create helpers for those and have them accept props so we don't have to declare almost the same boilerplate code in both tests, but I don't think this will help much. I'm more than happy to keep them as you suggested.

Reply:
The original test has a similar helper, but I find this more readable, and it only adds a few extra lines.

Reply:
Agreed 👍